modules/ROOT/pages/create-waf-policy.adoc

```diff
@@ -0,0 +1,77 @@
+. Enter a name for the policy and click *Save*. +
+The policy appears in the Security Policies list, where you can edit and delete policies.
+
+image::security-policy-list.png[]
+
+== Configure Request Rulesets
+
+Configure the rules to allow or block request traffic to your Web application. For more information about request rulesets, see xref:waf-rulesets#request_rule_sets[Request rulesets].
+
+. In the menu on the left, click *Request Rulesets*. +
+All the rules are disabled by default. For each rule, you can: +
+* *Disable ruleset* - (Default) Ruleset detection is turned off.
+* *Detect and allow violations* - The violation is detected and you will get information in your log at the INFO level, per incident.
+* *Detect and reject violations* - The request is rejected and returns a response status of `400 - BAD REQUEST`.
++
+When you hover over the `i` to the right of each rule, the rule ID range for that rule is displayed. This information can be used for testing rulesets and for disabling individual rules.
++
+image::waf-rule-set.png[]
++
+. In *Advanced performance options*, you can select *Disable body scanner*. If you do not check this option, the request body is scanned.
++
+[NOTE]
+Leaving the *Disable body scanner* option unchecked can be costly in terms of resources (CPU) depending on the size of the request messages. If request body checking is enabled, the CAP policy “Maximum Message Size” should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting the resources.
+. Click *Save*.
+
+== Configure Response Rulesets
+
+Configure the rules to allow or block responses to your Web application. For more information about response rulesets, see xref:waf-rulesets#response_rule_sets[Response rulesets]
+
+. In the menu on the left, click *Response Rulesets*. +
+All the rulesets are disabled by default. For each rule, you can: +
+* *Disable ruleset* - (Default) Ruleset detection is turned off.
+* *Detect and allow violations* - The violation is detected and you will get information in your log at the INFO level, per incident.
+* *Detect and reject violations* - The request is rejected and returns a response status of `400 - BAD REQUEST`.
+. In *Advanced Performance Options*, select *Disable body scanner* to improve performance. +
++
+[NOTE]
+Leaving the *Disable body scanner* option unchecked can be costly in terms of resources (CPU) depending on the size of the request messages. If request body checking is enabled, the CAP policy “Maximum Message Size” should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting the resources.
++
+The *Disable body scanner* works the same as it does for the request ruleset, however, there are some additional important caveats:
++
+[NOTE]
+Do not leave the *Disable body scanner* option unchecked if responses will exceed 512 KB.
++
```

Comment:
Sentence is confusing. Reword to: Check *Disable body scanner* if responses will exceed 512 KB.